pkg/kube/translation/apisix_route.go

```diff
@@ -144,6 +144,8 @@ func (t *translator) translateHTTPRouteV2beta2(ctx *TranslateContext, ar *config
 				pluginMap["key-auth"] = part.Authentication.KeyAuth
 			case "basicAuth":
 				pluginMap["basic-auth"] = make(map[string]interface{})
+			case "wolfRbac":
+				pluginMap["wolf-rbac"] = make(map[string]interface{})
```

Comment:
`v2beta2` has been deprecated, we will not add any new features to it.

Please remove this.